lib/config/presets/internal/replacements.ts

```diff
@@ -11,7 +11,17 @@ export const presets: Record<string, Preset> = {
       'replacements:jade-to-pug',
       'replacements:joi-to-scoped',
       'replacements:joi-to-unscoped',
-      'replacements:material-ui-to-mui',
+      'replacements:material-ui/codemod-to-mui/codemod',
```

Comment:
shouldn't we use one preset for this? they need to be applied in one PR anyways.